microbenchmark/apic-ipi/apic_ipi.c

```diff
@@ -69,7 +69,7 @@ static int apic_ipi_init(void)
 	char name[KSYM_NAME_LEN];
 	int i;
 
-	if (!options) {
+	if (!options || options >= MAXNUMA) {
```

Comment:
MAXNUMA is the maximum count of NUMA nodes, I don't understand why you need compare options and MAXNUMA?